src/handlers/decision.rs

```diff
@@ -152,6 +162,119 @@ pub(super) async fn handle_command(
     }
 }
 
+fn build_status_comment(
+    history: &BTreeMap<String, Vec<UserStatus>>,
+    current: &BTreeMap<String, Option<UserStatus>>,
+) -> anyhow::Result<String> {
+    let mut comment = "| Team member | State |\n|-------------|-------|".to_owned();
+    for (user, statuses) in history {
+        let mut user_statuses = format!("\n| {} |", user);
+
+        // previous stasuses
+        for status in statuses {
+            let status_item = format!(" ~~{}~~ ", resolution_to_str(&status.resolution));
+            user_statuses.push_str(&status_item);
+        }
+
+        // current status
+        let current_status = current.get(user).unwrap();
+        let user_resolution;
+        match current_status {
```

Comment:
NIT: we can do `let user_resolution = match { ....` instead of assigning in each branch, or I'm wrong?